packages/create-docusaurus/bin/index.js

```diff
@@ -55,17 +46,19 @@ program
   )
   .description('Initialize website.')
   .action(
-    (
+    async (
```

Comment:
Since this function doesn't use `await`, you can either remove this, or refactor the body to use `await` instead. Either makes sense.

Reply:
Since dynamic import with async-await is not supported in 12, I removed this async keyword.